src/main/java/boss/TimeTask.java

```diff
@@ -0,0 +1,76 @@
+package boss;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+
+/**
+ * Represents an event associated with times!
+ */
+public class TimeTask extends Task {
```

Comment:
Is it necessary to create a TimeTask class? Or could you have Deadline and Event ineherit from Task? Could be an alternative way to do things that you can try!